app/main.py

```
@@ -0,0 +1,120 @@
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+)
+
+logger = logging.getLogger(__name__)
+
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Ruff (0.14.10)

Unused function argument: `app`

(ARG001)